src/main/java/fi/aalto/cs/intellij/activities/RequiredPluginsCheckerActivity.java

```diff
@@ -147,25 +148,16 @@ private void notifyAndSuggestPluginsEnabling(List<IdeaPluginDescriptor> descript
             + getPluginsNamesString(descriptors) + ".",
         NotificationType.WARNING);
 
-    notification.addAction(new NotificationAction(
-        "Enable the required plugin(s) (" + getPluginsNamesString(descriptors) + ").") {
-
-      /**
-       * Activate all the required plugins.
-       */
-      @Override
-      public void actionPerformed(@NotNull AnActionEvent e, @NotNull Notification notification) {
-        descriptors.forEach(descriptor -> Objects
-            .requireNonNull(PluginManager.getPlugin(descriptor.getPluginId())).setEnabled(true));
```

Comment:
**testing approach:** what happens inside here... basically a loop that performs some actions done by the framework, do we need to test this?